airdrop/airdrop.gno

```diff
@@ -12,8 +13,13 @@ var (
 	ErrInvalidClaimsCount  = errors.New("INVALID_CLAIMS_COUNT")
 	ErrFirstClaimIsNot64   = errors.New("FIRST_CLAIM_MUST_BE_MULT_64")
 	ErrInsufficientBalance = errors.New("INSUFFICIENT_BALANCE")
+	ErrRefundNotAllowed    = errors.New("refund not allowed")
+	ErrTooEarly            = errors.New("too early")
+	ErrNoBalance           = errors.New("no balance to refund")
 )
 
+var timeNow = time.Now
```

Comment:
typo??
1. time.Now() 
2. time.Now().Unix()

Reply:
it's valid syntax

Reply:
it's valid syntax. this variable can be use like function later.

```go
var timeNow = time.Now

func main() {
	println(timeNow().Unix())
}
```

Reply:
what an weird syntax :(